### DISCUSSION TITLE
Map huiswerk_les9 with files

### DISCUSSION THREAD

**Discussion by ElleWilte** [community/community#32676] [Other Feature Feedback, Questions, & Ideas]
The URL 'https://github.com/ElleWilte/huiswerkles9.git/huiswerk_les9/home.html' generates with GitHub the error-messsage '404 This is not the page you are looking for'. The file home.html is , together with other maps and html-files, placed in the map huiswerk_les9. Without Github the web-page works fine. Please help me. Thanking you, Elle Evenhuis

**Comment by airtower-luna** [comment]
What do you expect to see under that URL? It seems to be an invalid mix of the HTTPS Git URL and a path within the repository.
